test/game_teams_collection_test.rb

```diff
@@ -0,0 +1,23 @@
+require './test/test_helper'
+require './lib/game_teams_collection'
+
+class GameTeamsCollectionTest < Minitest::Test
+  def setup
+    @game_teams_collection = GameTeamsCollection.new("./data/game_teams.csv")
+    @game_teams = @game_teams_collection.game_teams_array.first
+  end
+
+  def test_it_exists
+    assert_instance_of GameTeamsCollection, @game_teams_collection
+  end
+
+  def test_it_has_attributes
+    assert_instance_of Array, @game_teams_collection.game_teams_array
+  end
+
+  def test_it_can_create_games_from_csv
+    assert_instance_of GameTeams, @game_teams
+    assert_equal "away", @game_teams.hoa
+    assert_equal "LOSS", @game_teams.result
```

Comment:
HOA and result should be all we need for iteration 2 stats

Reply:
Noted